templates/flavors/default/kubeadmControlPlane.yaml

```diff
@@ -36,25 +36,12 @@ spec:
     preKubeadmCommands:
       - /kubeadm-pre-init.sh '{{ ds.meta_data.label }}' "${KUBERNETES_VERSION}"
     clusterConfiguration:
-      etcd:
```

Comment:
do we want to keep this even if we don't have separate disks?

Reply:
Yeah I think that sounds good to me, I'll add a note in the etcd docs